src/main/java/org/umssdiplo/automationv01/core/managepage/Login/Login.java

```diff
@@ -7,20 +7,25 @@
 import org.umssdiplo.automationv01.core.utils.PropertyAccessor;
 
 public class Login extends BasePage {
-    @FindBy(name = "email")
+    @FindBy(name = "username")
     private WebElement usernameInputField;
 
     @FindBy(name = "password")
     private WebElement passwordInputField;
 
-    @FindBy(css = ".btn-primary.btn-block")
-    private WebElement loginBtn;
+    @FindBy(css = ".mat-button.ng-star-inserted")
+    private WebElement loginButton;
+
+    @FindBy(css = ".background-primary.text-floral-white.mat-button")
+    private WebElement loginAcceptButton;
 
     public void setCredentials() {
         String username = PropertyAccessor.getInstance().getUser();
```

Comment:
Slipt the following events that it necessary to do
Example

public void enterUsernameFromSystem(){
String username = PropertyAccessor.getInstance().getSHUser();
CommonEvents.setInputField(usernameInputField, username);
}

maybe you will need to validate with wrong values

public void enterUsername(String username){
CommonEvents.setInputField(usernameInputField, username);
}